docs/source/tutorials/configure_data.rst

```diff
@@ -185,6 +185,42 @@ Convert the raw data config to the data container.
                 })
             )
 
+4. TransformersDummyDataContainer:
+Convert the transformer dummy data config to the data container.
+
+.. tabs::
+    .. tab:: Config JSON
+
+        .. code-block:: json
+
+            {
+                "name": "transformers_dummy_data_config",
+                "type": "TransformersDummyDataContainer",
+                "load_dataset_config": {
+                    "params": {
+                        "model_name": "meta-llama/Llama-2-7b-hf"
```

Comment:
Do you need model_name ? "input_model" has that information so there is no need to duplicate it in the .json config.

Reply:
If it is the workflow-level json config, user do not need to provide, since befor start `olive engine`, olive will try to validate the input json config and try to fill-in the corresponding value.
https://github.com/microsoft/Olive/blob/d930b2c001961e7aac7e15c37502e147ee99d7b7/olive/workflows/run/config.py#L210

But if only data_config or user use the python api to create data_config, they have to provide `model_name`.
I added the comments here.
![image](https://github.com/microsoft/Olive/assets/13343117/4d40f785-d574-4823-95da-f7813ae5f86d)

I was thinking the example in `configure_data.rst` should be executable solely. Then I added the model_name.
Please let me know if you want to remove.

Reply:
Please remove. Thanks!